Library/Homebrew/test/support/helper/spec/shared_context/integration_test.rb

```diff
@@ -85,6 +85,7 @@ def brew(*args)
         "-W0",
         "-I", "#{HOMEBREW_LIBRARY_PATH}/test/support/lib",
         "-I", HOMEBREW_LIBRARY_PATH.to_s,
+        "-I", "#{HOMEBREW_LIBRARY_PATH}/cask/lib",
```

Comment:
What's this needed for? It'd be nice to instead move logic that's needed for a generalised `brew` integration test out of `cask/lib`.

Reply:
The generic stuff is already outside of `cask/lib`, but we cannot get around having access to the `Cask` class.

Reply:
It doesn't load code, it only adds this path to the `$LOAD_PATH`.